resources/css/pages/loginpage.css

```diff
@@ -0,0 +1,22 @@
+@import url('../components/colors.css');
```

Comment:
Is er een speciale reden waarom dit niet geïmporteerd wordt als de andere bestanden (`@import 'colors.css'`)

Reply:
Omdat hij niet in dezelfde map zit.